src/local/int_cvx.lean

```diff
@@ -0,0 +1,102 @@
+import data.real.basic
+import linear_algebra.basic
+import linear_algebra.finite_dimensional
+import category_theory.endomorphism
+import topology.algebra.module
+import topology.instances.real
+import analysis.convex.basic
+
+-- This is intended as a Lean roadmap for the `int_cvx` lemma, currently Lemma 1.5 of loops.tex.
+-- Rather than a "human readable" proof of a single lemma,
+-- it is a longer sequence of lemmas, facts, and constructions, for now all sorried.
+
+-- TODO everything in this file could be generalised to arbitrary affine spaces over ℝ.
+variables {V : Type*} [add_comm_group V] [vector_space ℝ V]
+
+/-- The dilation about `c` with scale factor `r`. -/
+def dilation (c : V) (r : ℝ) : V → V := sorry
```

Comment:
What do you think about using an `affine_space`, and define it as a homomorphism to affine self-maps?

Reply:
Yes! I haven't actually looked much at `affine_space`. Is it ready for this?

I'd put in a TODO a few lines higher suggesting this generalisation.

Reply:
I see. It really seems like we should redo everything about convexity for affine spaces rather than linear spaces, perhaps before proceeding with much else... You'd made a start on this?

Reply:
Yes, it seems to make sense to first work on the affine space part of mathlib.